config.lua

```diff
@@ -1,6 +1,6 @@
 Config = {}
 
-Config.HasSpawn = true and GetResourceState('qb-spawn') ~= 'missing' or false
+Config.HasSpawn = true and GetResourceState('qbx-spawn') ~= 'missing' or false
```

Comment:
```suggestion
Config.HasSpawn = GetResourceState('qbx-spawn') ~= 'missing'
```
Whilst you are at it, might as well correct the syntax